doc/release-notes.md

```diff
@@ -35,65 +28,117 @@ reindex or re-sync the whole chain.
+the peer. It will return one of six options. For more information, see the `getpeerinfo` help documentation.
+- `getpeerinfo`: now has additional `last_block` and `last_transaction` fields that return the UNIX epoch time
+of the last block and the last valid transaction received from each peer.
+- `getblockchaininfo`: now returns a new `time` field, that provides the chain tip time.
+- `fundrawtransaction` and `walletcreatefundedpsbt`: when used with the `lockUnspents` argument now lock manually
+selected coins, in addition to automatically selected coins. Note that locked coins are never used in automatic coin
+selection, but can still be manually selected.
+- `testmempoolaccept`: returns `vsize` and a `fee` object with the `base` fee if the transaction passes
+validation. The `testmempoolaccept` RPC now accepts multiple transactions (still experimental at the moment, API may be
+unstable).
+
+## Added RPCs
+
+- `submitchainlock`: allows the submission of a ChainLock signature. Note: This RPC is whitelisted for the Platform
+RPC user.
+- `getassetunlockstatuses`: allows fetching of Asset Unlock txs by their withdrawal index. The RPC accepts an array
+of indexes and an optional block height. The possible outcomes per each index are: `chainlocked`, `mined`, `mempooled`,
+`unknown`. Note: If a specific block height is passed on request, then only `chainlocked` and `unknown` outcomes are
+possible. This RPC is whitelisted for the Platform RPC user.
+- `quorum dkginfo`: returns information about DKGs: `active_dkgs` and `next_dkg`.
+- `getrawtransactionmulti`: allows requesting a batch of up to 100 raw transactions per request.
+
+## Docker Build System
+
+Modified the Dockerfile to exclude the `dash-qt` graphical wallet interface from the set of binaries copied to
+`/usr/local/bin` during the Docker image build process. This change streamlines the Docker image, making it more
+suitable for server environments and headless applications, where the graphical user interface of `dash-qt` is not
+required. The update enhances the Docker image's efficiency by reducing its size and minimizing unnecessary components.
 
-The miner has been disabled for Windows binaries built via Guix. This resolves
-an issue that was preventing users from installing v20.0.3 due to false
-positive alerts from Microsoft Defender and other antivirus providers.
+## Command-line options
 
-Use a self-signed Windows code signing certificate instead of expired one. The
-expired certificate being used previously caused most antivirus software to trigger.
-The new self-signed certificate results in a greatly improved installation experience.
+- The same ZeroMQ notification (e.g. `-zmqpubhashtx=address`) can now be specified multiple times to publish the same
+notification to different ZeroMQ sockets.
+- `-capturemessages`: Capture all P2P messages to disk, see contrib/message-capture/message-capture-docs.md for more info
```

Comment:
Is that supposed to be a link to the markdown file?

Reply:
Maybe this?
```suggestion
- `-capturemessages`: Capture all P2P messages to disk, see [contrib/message-capture/message-capture-docs.md](https://github.com/dashpay/dash/blob/v20.1.0/contrib/message-capture/message-capture-docs.md) for more info
```

Reply:
path can be relative so far as I know, such as `../contrib/message-capture/message-capture-docs.md`

+ `-capturemessages`: Capture all P2P messages to disk, see [message-capture-docs.md](contrib/message-capture/message-capture-docs.md) for more info

Reply:
That should work (once the tag exists) :+1: